pkg/network/tracer/connection/kprobe/tracer.go

```diff
@@ -174,6 +174,15 @@ func LoadTracer(cfg *config.Config, m *manager.Manager, mgrOpts manager.Options,
 	return closeFn, TracerTypePrebuilt, err
 }
 
+func perCPUSupported() bool {
+	kv, err := kernel.HostVersion()
+	if err != nil {
+		return false
+	}
+
+	return kv >= kernel.VersionCode(4, 7, 0)
```

Comment:
I am not sure I understand your question. Why would it not work especially on those systems?

Reply:
eBPF telemetry is turned off for kernels < 4.14. 
This is just so we don't end up using per-cpu maps when they are not available on older kernels when loading the ebpf programs.

Reply:
I guess the question is whether this check would return false on centos when per-cpu maps are available there? Could we use one of the checks here instead: https://github.com/cilium/ebpf/blob/main/features/map.go

Reply:
The check in Cilium ultimately just compares kernel version as well. https://github.com/cilium/ebpf/blob/main/features/map.go#L103

Did you on the otherhand mean to duplicate this functionality https://github.com/cilium/ebpf/blob/main/features/map.go#L79 in out codebase ?

Reply:
That is a little disappointing. Maybe centos/redhat don't support cpu maps at all, so that is why it is a just a version comparison? I don't think we should duplicate it, but still think it is a good idea to use it.